core/src/main/kotlin/com/justai/jaicf/builder/ScenarioBuilder.kt

```diff
@@ -276,7 +279,21 @@ abstract class ScenarioBuilder(
         fun intent(intent: String) = add(Transition(
             fromState,
             toState,
-            IntentActivationRule(intent)
+            IntentByNameActivationRule(intent)
+        ))
+
+        /**
+         * Appends any-intent activator to this state. Means that any intent can activate this state.
+         * Requires a [com.justai.jaicf.activator.intent.IntentActivator] in the activators' list of your [com.justai.jaicf.api.BotApi] instance.
+         *
+         * @see com.justai.jaicf.activator.intent.IntentActivator
+         * @see com.justai.jaicf.api.BotApi
+         */
+        fun anyIntent() = add(
```

Comment:
I think we need to make anyIntent internal and provide like-fallback builder:
```kotlin
    fun anyIntent(
        state: String = "anyIntent",
        action: ActionContext.() -> Unit
    ) = state(
        name = state,
        noContext = true,
        body = {
            activators { anyIntent() }
            action(action)
        }
    )
```

to use something like

```kotlin
        state("/someIntent") {
           ...
        }

        anyIntent {
            activator.caila?.topIntent?.answer?.let { 
                reactions.say(it)
            }
        }
```

Reply:
Great idea!
Can you propose a name for this "state-like" builder function? I don't want both methods (builder and rule) to have the same name.

Reply:
We can use `intent()` without argument as _any-intent_ rule, and `anyIntent {}` as state builder. What do you think?

Reply:
Maybe we just make `anyIntent()` rule internal, therefore only builder will be available in scenario?
I see no point making rule public, this will only give user way to write bad code, giving not so much of benefits.

Reply:
```kotlin
intent() {
    .. anyIntent logics
}
```
looks a bit counter-intuitive, and also I think we'll be adding more builders to scenario. For example, a builder for state with single intent activator (it looks to be a fairly common use case). We'd better to keep this method signature reserved.
``` kotlin
fun intent(
        intent: String,
        action: ActionContext.() -> Unit
    ) = state(
        name = intent,
        noContext = true,
        body = {
            activators { intent(intent) }
            action(action)
        }
    )
```
to optimise scenario building with states with only one intent activator (i think that's most of the cases). E.g.:
```kotlin
        intent("Hello) {
            reactions.say("Hi there!")
        }
```